src/main/kotlin/com/project/twiliospring/controller/RestLoginController.kt

```diff
@@ -0,0 +1,94 @@
+package com.project.twiliospring.controller
+
+import com.project.twiliospring.domain.LoginResult
+import com.project.twiliospring.domain.User
+import com.project.twiliospring.domain.dto.F2aCodeDTO
+import com.project.twiliospring.domain.dto.LoginDTO
+import com.project.twiliospring.exception.F2ARecordExpiredException
+import com.project.twiliospring.service.LoginService
+import com.project.twiliospring.util.HTTPUtil
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/login")
+class RestLoginController(
+    val loginService: LoginService
+) {
+    @PostMapping
+    fun login(
+        @RequestBody loginDTO: LoginDTO,
+        response: HttpServletResponse
+    ): ResponseEntity<Any> {
```

Comment:
It is a bad practice to return different response formats from controller. Please structures it